lib/src/internal/constants.dart

```diff
@@ -3,6 +3,7 @@
 // in the LICENSE file.
 
 import '../types/thumbnail.dart';
+import '../types/types.dart';
```

Comment:
This is redundant. Docs will inject type references even if you didn't import them.

Reply:
In the IDEA/VSCode, if I not import the dart, I can't cmd+click to jump the source to the `RequestType` to the file of defined.